src/sentry/options/defaults.py

```diff
@@ -1333,6 +1333,12 @@
     default=0,
     flags=FLAG_PRIORITIZE_DISK | FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "project-abuse-quota.log-limit",
+    type=Int,
+    default=0,
```

Comment:
This doesn't actually stop ingestion while it's rolling out.

Some option magic going on here:
https://github.com/getsentry/sentry/blob/28330fbb10c3dc352c9d28884e76534f612666a0/src/sentry/quotas/base.py#L250-L256